.github/workflows/run-test.yml

```diff
@@ -36,14 +38,30 @@ jobs:
         uses: ./.github/actions/prepare-android-env
 
       - name: Run test
-        run: ./gradlew test
```

Comment:
Question: Why is this removed?

Reply:
it is replaced by a task that I created codeCoverage